docs/quick-start-demo.md

```diff
@@ -15,98 +15,133 @@ keywords:
+1. Navigate to <a href="https://issuer-demo.privado.id/">the Issuer Node UI testing environment</a>.
 
    :::warning
 
-   This Issuer Node is publicly available and used only for testing purposes. Do not use personal or sensitive data. All data is deleted every 48 hours.
+   Only use this publicly available Issuer Node for testing purposes. Do not submit personal or sensitive data. All data is deleted every 48 hours.
 
    :::
 
-2. Now you need to import the schema. Click on **Import Schema** and paste our previously generated schema IPFS address `ipfs://QmTSwnuCB9grYMB2z5EKXDagfChurK5MiMCS6efrRbsyVX`:
+2. Next, import the schema. 
+
+    2.1 Click on **Import Schema** and paste the schema's IPFS address `ipfs://QmTSwnuCB9grYMB2z5EKXDagfChurK5MiMCS6efrRbsyVX`:
 
-   <div align="center">
-       <img width="100%" src={useBaseUrl("img/quick-start-demo/import-schema.png")}></img>
-   </div>
+       <div align="center">
+           <img width="100%" src={useBaseUrl("img/quick-start-demo/import-schema.png")}></img>
+       </div>
 
-   You may preview the schema and then Import it.
+    2.2 Preview the schema and then Import it.
 
-3. You can go ahead and click on **Issue Credential** in the top righ-hand corner. Choose **Credential Link** on the next page and your schema on the dropdown menu ("POAP01", in our case). For this credential, we are providing a proof of attendance to an event in Paris:
+3. Next, click on **Issue Credential** in the top right-hand corner. Choose **Credential Link** on the next page and your schema on the dropdown menu ("POAP01", in our case). For this credential, we are providing a proof of attendance at an event in Paris:
 
 <div align="center">
     <img width="500" src={useBaseUrl("img/quick-start-demo/create-credential.png")}></img>
 </div>
 
-4. After you click on **Create Credential Link**, you'll be presented with a Universal Link and a Deep Link to the credential offer, along with a QR code configured for these links. The QR code can be scanned directly with the Privado ID Wallet app.
+4. Click on **Create Credential Link** to be presented with a Universal Link and a Deep Link to the credential offer, along with a QR code configured for these links. The QR code can be scanned directly with the Privado ID Wallet app, that is, the Mobile App.
 
     :::info
 
-    When the user interacts with the [<ins>Universal Link</ins>](/docs/wallet/universal-links.md) , it will launch the Privado ID Web Wallet in the browser or Privado ID Wallet app in case of a mobile phone displaying the credential offer to claim the credential. Deep link could be handled only by mobile wallet app though.
+    When the user interacts with the [<ins>Universal Link</ins>](/docs/wallet/universal-links.md), it will launch the Privado ID Web Wallet in the browser or Privado ID Wallet app in case of a mobile phone displaying the credential offer to claim the credential. The Deep link supports the Mobile Wallet App.
```

Comment:
> The Deep link supports the Mobile Wallet App.

Here we lose the concept that Deep Links can only be used from mobile applications and not in the Web Wallet.

Reply:
I will try again to simplify this, but retain the data

Reply:
PTAL @ lines 144-148 that now cover this @amonsosanz WDYT?

Reply:
Not yet precise I'd say...

`Deep Link supports the Mobile Wallet App only` should be:
`Deep Links are supported by the Privado ID Mobile App Wallet and other mobile wallets`

* Deep links do not support wallets, but the other way around.
* Not only our mobile wallet can use these, but any mobile wallet.

`The Universal Link supports the Web App only` should be
`The Universal Links are supported by both the Web Wallet and the Privado ID Mobile App Wallet. They might also be supported by third-party mobile wallets`

* Universal links do not support wallets, but the other way around.
* Our web and mobile wallets and other mobile wallets might also support these.

All this is a bit tricky 😅

Let's also make sure I'm right checking it with @0xpulkit.

Reply:
Thanks @amonsosanz did another pass PTAL.

Reply:
Now it's perfect! Thx to you.